File: app/src/main/java/com/neome/java/api/ent/entMain/sig/SigSpreadsheetRowRemove.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.entMain.sig;

import com.neome.java.api.meta.base.Types.RowId;
import com.neome.java.api.meta.base.dto.EnvValidationError;
import com.neome.java.api.nucleus.base.sig.Sig;

import java.util.Map;

@SuppressWarnings("unused")
public class SigSpreadsheetRowRemove extends Sig
{
  public Map<RowId, EnvValidationError> errorMap;
}

File: app/src/main/java/com/neome/api/ent/entMain/sig/SigSpreadsheetRowRemove.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entMain.sig

import com.neome.api.meta.base.dto.EnvValidationError
import java.util.Map
import com.neome.api.meta.base.Types.RowId
import com.neome.api.nucleus.base.sig.Sig

interface SigSpreadsheetRowRemove : Sig
{
  val errorMap: Map<RowId, EnvValidationError>
}